code/public/index.html

```diff
@@ -1,34 +1,33 @@
 <!DOCTYPE html>
 <html lang="en">
+	<head>
+		<meta charset="utf-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1" />
+		<meta
+			name="description"
+			content="Twitter-like app, Technigo students can share their positive energy, created with React."
+		/>
+		<meta name="keywords" content="app, React" />
```

Comment:
Awesome with meta-tags! Do you need to check indentation for consistency from here to line 12 or maybe it's ok when it's so short to have two attributes on the same line?

Reply:
Hi, I have changed my indentation, everything looked ok in VS code but not in Git, I hope this is changed for future project now 😄